result/block-scoped-var.js

```diff
@@ -0,0 +1,57 @@
+/*eslint block-scoped-var: "error"*/
+
+function doIf() {
+    if (true) {
+        var build = true;
+    }
+
+    console.log(build);
+}
+
+function doIfElse() {
+    if (true) {
+        var build = true;
+    } else {
+        var build = false;
+    }
+}
+
+function doTryCatch() {
+    try {
+        var build = 1;
+    } catch (e) {
+        var f = build;
```

Comment:
__[development][eslint]__
* Error - 'f' is defined but never used (no-unused-vars)
* Error - 'build' used outside of binding context. (block-scoped-var)